x-pack/test/api_integration/deployment_agnostic/services/synthetics_monitor.ts

```diff
@@ -0,0 +1,243 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { RoleCredentials } from '@kbn/ftr-common-functional-services';
+import { SYNTHETICS_API_URLS } from '@kbn/synthetics-plugin/common/constants';
+import { syntheticsMonitorType } from '@kbn/synthetics-plugin/common/types/saved_objects';
+import { EncryptedSyntheticsSavedMonitor } from '@kbn/synthetics-plugin/common/runtime_types';
+import { MonitorInspectResponse } from '@kbn/synthetics-plugin/public/apps/synthetics/state/monitor_management/api';
+import { v4 as uuidv4 } from 'uuid';
+import expect from '@kbn/expect';
+import { ProjectAPIKeyResponse } from '@kbn/synthetics-plugin/server/routes/monitor_cruds/get_api_key';
+import moment from 'moment/moment';
+import { omit } from 'lodash';
+import { KibanaSupertestProvider } from '@kbn/ftr-common-functional-services';
+import { DeploymentAgnosticFtrProviderContext } from '../ftr_provider_context';
+
+// fix saml auth type
```

Comment:
I thought I fixed this. I'll have to adjust.

Reply:
Do you want this comment here?

Reply:
This is also outdated. I believe you're looking at an odd commit.